shared/src/dto/vehicle/vehicle.dto.ts

```diff
@@ -0,0 +1,105 @@
+import { ApiProperty, PartialType } from '@nestjs/swagger';
+import { IsNotEmpty, IsOptional, ValidateNested } from 'class-validator';
+import { Type } from 'class-transformer';
+
+export class DtoVehicle {
+  @ApiProperty()
+  @IsNotEmpty()
+  id: number;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  name: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  licensePlate: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  vin: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Please remove this comment and change the type to `ApiDate` (as suggested by the ESLint rule)